src/shapes/polyline.class.ts

```diff
@@ -7,11 +7,25 @@ import { IPoint, Point } from '../point.class';
 import { TClassProperties } from '../typedefs';
 import { classRegistry } from '../util/class_registry';
 import { makeBoundingBoxFromPoints } from '../util/misc/boundingBoxFromPoints';
+import { calcDimensionsMatrix, transformPoint } from '../util/misc/matrix';
 import { projectStrokeOnPoints } from '../util/misc/projectStroke';
+import { TProjectStrokeOnPointsOptions } from '../util/misc/projectStroke/types';
 import { degreesToRadians } from '../util/misc/radiansDegreesConversion';
 import { toFixed } from '../util/misc/toFixed';
 import { FabricObject, fabricObjectDefaultValues } from './Object/FabricObject';
 
+const strokeProjectionOptions: (keyof TProjectStrokeOnPointsOptions)[] = [
```

Comment:
I forgot what this is.
Shouldn't it be covered by `strokeBBoxAffectingProperties`?
The only diff is the scale keys...
I need help to recall the difference if any

Reply:
Maybe not...
Is not a big deal anyways

Reply:
Are you suggesting to use `strokeBBoxAffectingProperties` instead of `strokeProjectionOptions` in `_getTransformedDimensions` function?

```diff
     if ( 
       Object.keys(options).some((key) => 
+       (
+               strokeBBoxAffectingProperties.includes( 
+                      key as keyof TProjectStrokeOnPointsOptions 
+               ) 
+              || this.strokeUniform
+       )
       ) 
     ) {

     if ( 
       Object.keys(options).some((key) => 
-         strokeProjectionOptions.includes( 
-          key as keyof TProjectStrokeOnPointsOptions 
         ) 
       ) 
     ) {
```

Seems to make sense to me and avoids recalculating the projections when `strokeUniform = false` with scaling.

Reply:
Perhaps
I just saw it when I reviewed to merge I did and didn't understand why there are 2 similar objects that serve for a very similar purpose. So I thought I messed up the merge commit...

Reply:
https://github.com/fabricjs/fabric.js/pull/8374/commits/494a10ee2f8c2278ae9a55b20bf50cf6ee25b064